### PR TITLE
fix: allow selecting only value on double click on DescriptionsItem

### LINKED ISSUE #24977: Double-clicking the value of `Descriptions.Item` selects both label and value
Jun-14-2020 02-34-35

- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://ant.design/components/descriptions/](https://ant.design/components/descriptions/)

### Steps to reproduce
Double-Click the value of `Descriptions.Item`

### What is expected?
Only the value should be selected.

### What is actually happening?
Both label and value are selected.

| Environment | Info |
|---|---|
| antd | 4.3.3 |
| React | - |
| System | macOS Mojave |
| Browser | Chrome 83.0.4103.61 (Official Build) (64-bit) |

This is not harmful at all, but a user normally would expect that only the value is selected.

### PR DESCRIPTION
**Pull request by harupy**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Perfermance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

Fix #24977

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by Rustin-Liu** [comment]
The CI broken.

File: components/descriptions/Cell.tsx [hunk 1]
```diff
@@ -63,7 +63,11 @@ const Cell: React.FC<CellProps> = ({
           {label}
         </span>
       )}
-      {content && <span className={classNames(`${itemPrefixCls}-item-content`)}>{content}</span>}
+      {content && (
+        <span className={classNames(`${itemPrefixCls}-item-content`)}>
+          <div>{content}</div>
```
Comment (Rustin-Liu) [inline components/descriptions/Cell.tsx:68]: I don't think put a div into span is a good idea.

Comment (Rustin-Liu) [reply components/descriptions/Cell.tsx:68]: Maybe we should consider other element?

Comment (harupy) [reply components/descriptions/Cell.tsx:68]: @Rustin-Liu Thanks for the comment.

I agree. I found a related issue on SO:

https://stackoverflow.com/questions/43690276/text-selection-inside-inline-blocks-without-space-between

Comment (harupy) [reply components/descriptions/Cell.tsx:68]: @Rustin-Liu What do you think?

Comment (Rustin-Liu) [reply components/descriptions/Cell.tsx:68]: It's OK for me. Thanks!

**Comment by afc163** [comment]
Could we try this?

```diff
.ant-descriptions-item>span {
- display: inline-block;
+ display: inline-flex;
+ align-items: baseline;
}
```

**Comment by harupy** [comment]
@afc163 Got it.

**Comment by harupy** [comment]
@afc163 Verified it works (tested on my local).

Jun-16-2020 14-43-53

**Comment by harupy** [comment]
@Rustin-Liu @afc163 Thanks for the support!

**Pull request merged**